packages/elements/src/tree-select/index.ts

```diff
@@ -284,7 +284,7 @@ export class TreeSelect extends ComboBox<TreeSelectDataItem> {
    * @returns Is all selected
    */
   protected get allSelected (): boolean {
-    return this.memo.selected === this.memo.selectable;
+    return this.memo.selectable > 0 && this.memo.selected === this.memo.selectable;
```

Comment:
Can we create a new boolean variable to store this condition because it has been use in many places ?